cfme/tests/infrastructure/test_set_default_filter.py

```diff
@@ -73,7 +73,7 @@ def test_clear_datastore_filter_results(appliance):
     """ Test for clearing filter results for datastores."""
     dc = DatastoreCollection(appliance)
     view = navigate_to(dc, 'All')
-    view.sidebar.datastores.tree.click_path('Datastores', 'All Datastores', 'Global Filters',
+    view.sidebar.datastores.tree.click_path('All Datastores', 'Global Filters',
```

Comment:
Is it correct for all versions of CFME?

Reply:
yes, this is bug created during web_ui->widgetastic conversion